### PR TITLE
ENCD-4388 excluding not relevant experiments form RE audits

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: src/encoded/audit/reference_epigenome.py [hunk 1]
```diff
@@ -97,15 +97,16 @@ def audit_reference_epigenome_assay_types_requirments(value, system):
         project_detail = 'required according to standards of Minimal IHEC Reference Epigenome.'
```
Comment (caseylitton) [inline src/encoded/audit/reference_epigenome.py]: Rebase to get tests passing.

Comment (gabdank) [reply src/encoded/audit/reference_epigenome.py]: rebased

Comment (caseylitton) [reply src/encoded/audit/reference_epigenome.py]: thx, one test failing.  could be related to update?

src/encoded/tests/test_audit_reference_epigenome.py::test_reference_epigenome_with_required_assays FAILED

**Comment by keenangraham** [comment]
It looks like `test_reference_epigenome_with_required_assays` still needs to be fixed and branch rebased.

**Pull request merged**